core/templates/pages/topic-editor-page/modal-templates/questions-list-select-skill-and-difficulty-modal.component.html

```diff
@@ -68,10 +68,11 @@ <h3 *ngIf="linkedSkillsWithDifficulty.length > 0">
       You can only set difficulty after selecting skills linked to the question. Please go back and link skill(s) to this question.
     </div>
     <div class="list-group">
-      <div class="list-group-item"  *ngFor="let skillWithDifficulty of linkedSkillsWithDifficulty; trackBy: index">
+      <div class="list-group-item"  *ngFor="let skillWithDifficulty of linkedSkillsWithDifficulty; let idx = index;">
         <label class="list-group-item-heading skill-description">{{ skillWithDifficulty.getDescription() }}</label>
         <oppia-question-difficulty-selector [skillIdToRubricsObject]="skillIdToRubricsObject"
-                                            [(skillWithDifficulty)]="skillWithDifficulty">
+                                            [skillWithDifficulty]="skillWithDifficulty"
+                                            (skillWithDifficultyChange)="changeSkillWithDifficulty($event, idx)">
```

Comment:
Shouldn't skillWithDifficultyChange start with a verb instead (if it's an action)?